appveyor.yml

```diff
@@ -38,9 +38,10 @@ install:
   - call set ENVIRONMENT_NAME=%%ENVIRONMENT_NAME:PLATFORM=%platform%%%
   - call set ENVIRONMENT_NAME=%%ENVIRONMENT_NAME:CONFIGURATION=%configuration%%%
   - cd %TEMP%
-  - echo *** Patching NSIS
-  - cd %APPVEYOR_BUILD_FOLDER%
-  - patch "C:\Program Files (x86)\NSIS\Include\MultiUser.nsh" < build\appveyor\NSIS.patch
+  # Not needed anymore since we switched to wix
+  #- echo *** Patching NSIS
+  #- cd %APPVEYOR_BUILD_FOLDER%
+  #- patch "C:\Program Files (x86)\NSIS\Include\MultiUser.nsh" < build\appveyor\NSIS.patch
```

Comment:
Please remove these lines instead of commenting them out.

Reply:
I don't think it's a good idea to remove them since we did not alredy remove the build/nsis directory already. I think we should keep it in the 2.1 branch and remove these lines + the build/nsis directory in the master branch.

I will update this PR later

Reply:
Why keep the the nsis directory in the 2.1 branch when it will not be used?

Reply:
to be able to switch back easily after the first user feedbacks ;)